sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_operationid.json

```diff
@@ -563,7 +563,7 @@
             "get": {
                 "summary": "Gets the status of all import/export jobs in an iot hub",
                 "description": "Gets the status of all import/export jobs in an iot hub. See https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities for more information. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
-                "operationId": "JobClient_GetImportExportJobs",
+                "operationId": "Jobs_GetImportExportJobs",
```

Comment:
We decided the type name *Client (e.g. JobClient) was fine, right? It is just the property name on the main CL client that we'd strip the "Client" suffix.

Reply:
Yes right. This is for generated code, which already adds a suffix Client/RestClient for the first part . Does that answer your question?